server/eclipse-project/src/main/java/us/freeandfair/corla/json/DoSDashboardRefreshResponse.java

```diff
@@ -111,20 +112,25 @@ public class DoSDashboardRefreshResponse {
    */
   private final SortedMap<Long, AuditType> my_audit_types;
 
+  /**
+   * The generate assertions summaries, for IRV contests. Keyed by contest name (which is repeated
+   * in the GenerateAssertionsSummary).
+   */
+  private final List<GenerateAssertionsSummary> my_generate_assertions_summaries;
+
   /**
    * Constructs a new DosDashboardRefreshResponse.
    *
```

Comment:
It looks like the parameters in the comments don't really match those in the method.

Reply:
Now fixed.